src/rpc/RPCHelpers.cpp

```diff
@@ -193,8 +193,7 @@ getAccount(
         return {};
     }
 
-    return Status{
-        RippledError::rpcACT_MALFORMED, field.to_string() + "Malformed"};
+    return Status{RippledError::rpcACT_NOT_FOUND};
```

Comment:
I don't think we should make this change. The account is malformed, as opposed to well formed but not found. This is the issue we were talking about before, where rippled interprets malformed accounts as just strings and treats them as a seed. Given that we decided to not accept seeds as accounts, I believe we are already returning the right error here, and there is no change that needs to be made to the code as is.